### PR TITLE
VTCCA-3207: appoint agent to some properties

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/controllers/agent/ManageAgentController.scala [hunk 1]
```diff
@@ -150,4 +151,16 @@ class ManageAgentController @Inject()(
...
 
+  private def joinOldJourney(agentCode: Long) =
+    Redirect(
+      controllers.agentAppointment.routes.AppointAgentController.getMyOrganisationPropertyLinksWithAgentFiltering(
+        pagination = PaginationParameters(page = 1, pageSize = 15),
+        params = GetPropertyLinksParameters(
+          sortfield = ExternalPropertyLinkManagementSortField.ADDRESS,
+          sortorder = ExternalPropertyLinkManagementSortOrder.ASC),
+        agentCode = agentCode,
+        agentAppointed = Some(Both.name),
```
Comment (PeterPerhac) [inline app/controllers/agent/ManageAgentController.scala]: what is agentAppointed = Both?

**Comment by PeterPerhac** [comment]
👍 Looks good to me. Only had one question

**Comment by darrell-plant** [comment]
LGTM 👍 (apart from Pete's question)

File: app/controllers/agent/ManageAgentController.scala (hunk 1 above)
Comment (habeeb01) [reply app/controllers/agent/ManageAgentController.scala]: passing "agentAppointed = Both" as a param basically means that you are requesting all property links (from that org) regardless of whether they have an agent appointed

**Pull request merged**